src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -33,6 +33,54 @@ enum SueStatus {
   WAIT_THIRDPARTY = 'TICKET_STATUS_WAIT_THIRDPARTY'
 }
 
+export const useReverseGeocode = () => {
```

Comment:
please move this in an own hook file

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1129/commits/1e3376ec8289d89335d6508806e92b69225ae9f9